gr-blocks/lib/file_source_impl.cc

```diff
@@ -86,12 +89,32 @@ namespace gr {
     bool
     file_source_impl::seek(long seek_point, int whence)
     {
-      return fseek((FILE*)d_fp, seek_point *d_itemsize, whence) == 0;
+      seek_point += d_start_offset_items;
+
+      if (whence == SEEK_SET)
```

Comment:
I'm a fan of using braces throughout the if/else clause if use them anywhere in a if/else clause. I know the GNU Radio Project has no strict guidelines for this.